packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -326,6 +327,9 @@ const BlockInspectorSingleBlock = ( { clientId, blockName } ) => {
 				className={ blockInformation.isSynced && 'is-synced' }
 			/>
 			<BlockVariationTransforms blockClientId={ clientId } />
+			{ blockName === 'core/query' && (
```

Comment:
Can we just add a comment, that this should leave with the first chance we get(either relocation of CreateNewPostLink inside Query or any other way)?